pages/searching/time_frame_selector.rst

```diff
@@ -1,38 +1,96 @@
+hit the search button. The selector offers a wide set of relative time frames that fit most of your 
+search needs, including an *All Time* option.
+
+.. image:: /images/searching/relative_time_frame_revised.png
+
+Consider how this filter works:
+
+* The *From* field allows you to type in values and select units for time via a dropdown menu. 
+  You can choose from seconds, minutes, hours, and days. For your convenience, you can click the *Preset Times* 
+  button to access pre-detemined times, interpreted in minutes, hours, and days. If you decide to select all 
+  messages instead, your dashboard would display data from the data of first ingestion.
+  
+* The *Until* date allows relative time ranges to end at a specific period instead of default to the current time/date.
 
 Absolute time frame selector
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-When you know exactly the boundaries of your search, you want to use the absolute time frame selector. Simply introduce the dates and
-times for the search manually or click in the input field to open up a calendar where you can choose the day with your mouse.
+
+When you know precisely the boundaries of your search, use the absolute time frame selector. This filter 
+provides an accordion to separate the Calendar from Timestamp inputs.
+
+This option displays an accordion containing two options:
+
+* Calendar
+* Timestamp
+
+In the Calendar option, use the hourglass icon to jump from the very beginning of the day (00:00:00.000) to 
+the very end of the day (23:59:59.99).
+
+To understand, Calender in more detail, consider the functions of *Until* and *From* on the Calendar option:
+
+* *Until* defaults to disabling all dates previous to the selected *From* date
+* *From* date will disable all previous dates if you configure a *Query Time Range Limit* (in the *System > Configurations* page)
+
+.. image:: /images/searching/absolute_selector_calendar.png
+
+You can use the magic wand icon for both Calendar and Timestamp.
+
+* In *Calendar*, the icon updates the *Time* to the current time but does not modify the date in the calendar
+* In *Timestamp*, the icon updates the entire *Timestamp* to the current date and time.
+
+.. image:: /images/searching/absolute_selector_timestamp.png
+
 
 Keyword time frame selector
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Graylog offers a keyword time frame selector that allows you to specify the time frame for the search in natural language like *last hour* or *last 90 days*. The web interface shows a preview of the two actual timestamps that will be used for the search.
+Graylog offers a keyword time frame selector that allows you to specify the time frame for the search in natural 
+language like *last hour* or *last 90 days*. The web interface shows a preview of the two actual timestamps that 
+will be used for the search.
 
-.. image:: /images/searching/queries_keyword_time_selector.png
+.. image:: /images/searching/keyword_time_selector.png
```

Comment:
This image is not found. It looks like the rename might not have been committed.

![image](https://user-images.githubusercontent.com/3423655/123157869-b1b6d480-d430-11eb-8656-80ebfae7c16a.png)

Reply:
Oh no.

Reply:
Fixed.